packages/strapi-plugin-upload/admin/src/components/CardControl/Wrapper.js

```diff
@@ -21,6 +22,31 @@ const Wrapper = styled.div`
     `
     transform: rotate(90deg)
   `};
+
+  &:hover {
+    ${({ type }) => {
+      if (type === 'trash-alt' || type === 'clear') {
+        return `
+          background-color: #FFA784;
```

Comment:
All this file colors are set in the theme/colors.js except this one (which is used several time in the content-manager) so can you please replace all this files colors with a theme and add this one in the colors.js (maybe calling it mediumOrange) if it won't take you too long time?